distributed/tests/test_ssl.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function, division, absolute_import
+
+import subprocess
+
+from distributed import (Client, Scheduler, Worker)
+from distributed.utils_test import gen_cluster, inc, loop
+
+import pytest
+import os
+
+@pytest.fixture(scope="function")
+def ssl_kwargs(tmpdir):
+
+    ssl_dir = tmpdir.mkdir("ssl")
+    dirname = ssl_dir.dirname
+
+    keyfile = os.path.join(dirname, 'test.key')
+    certfile = os.path.join(dirname, 'test.pem')
+
+    p = subprocess.Popen([
+        'openssl', 'req',
```

Comment:
Running this at each test invocation makes it mandatory to have OpenSSL installed to run the tests, right?
If so, I'd rather have cert and key files added to the tests directory, and a small script added to regenerate them when needed.
(OpenSSL isn't usually installed on Windows, for instance)